opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbankingapi/mapper/FacadeToRestMapperTest.java

```diff
@@ -0,0 +1,33 @@
+package de.adorsys.opba.tppbankingapi.mapper;
+
+import de.adorsys.opba.protocol.api.dto.result.body.AccountListBody;
+import de.adorsys.opba.protocol.api.dto.result.body.AccountListDetailBody;
+import de.adorsys.opba.tppbankingapi.ais.model.generated.AccountList;
+import de.adorsys.opba.tppbankingapi.mapper.generated.AccountListFacadeToRestMapperImpl;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class FacadeToRestMapperTest {
```

Comment:
I suggest using JSON fixture to map those classes - read input and expectation from resources folder and validate results by field comparison.